gammapy/spectrum/observation.py

```diff
@@ -705,6 +718,39 @@ def show_obs(idx):
         return interact(show_obs, idx=(0, max_, 1))
 
 
+    def to_spectrum_datasets(self, model=None, fit_range=None, forward_folded=True):
```

Comment:
This will likely go if `SpectrumExtract` no longer returns a `SpectrumObservationList` but simply a list of `SpectrumDatasetOnOff`.